umh.docs.umh.app/content/en/docs/features/postgresql_and_grafana_alerts.md

```diff
@@ -0,0 +1,245 @@
+          fired.
+   
+     4. In section four, you can add details and descriptions for your rule.
+     
+     5. In section five, you are required to assign labels to your alert, 
+        ensuring it is directed to the appropriate contacts. For example,
+        you may designate a label **team** with **alertrule1**: `team = operator`
+        and **alertrule2:** `team = management`. It can be helpful to use
+        labels more than once, like  **alertrule3:** `team = operator`, to link 
+        multiple alerts to a contact point at once.
+     
+     Your rule is now completed; click on **Save and Exit** on the right upper 
+     corner, next to section one.
+
+3. **Create a Contact Point**
+
+     A contact point consolidates addresses and services to be notified. When a
+     linked alert is triggered, everyone within the contact point receives a 
+     message. To create a contact point, navigate to **Contact points**, located
+     at the top of the Grafana alerting page.
+     - Click on the blue **+ Add contact point** button.
+     - Choose a name for your contact point.
+     - Pick the receiving service; in this example, Discord.
+     - Generate a new Webhook in your Discord server (Server Settings → 
+       Integrations → View Webhooks → New Webhook or create Webhook). 
+       Assign a name to the Webhook and designate the messaging channel.
+       Copy the Webhook URL from Discord and insert it into the corresponding 
+       field in Grafana. Customize the message to Discord under 
+       **Optional Discord settings** if desired.
+     - If needed, add more services to the contact point, by clicking **+ Add 
+       contact point integration**.
+     - Save the contact point; you can see it in the **Contact points** list,
+       below the **grafana-default-email** contact point.
+   
+   The integration of alerting by mail is very complicated to set up in Grafana
+   and therefore not part of this tutorial.
+
+4. **Configure Notification Policies**
+
+     A notification policy page allows you to link alerts with a contact 
+     point. Notification policies can be added in the **Notification policies**
+     section in the Grafana alerting page, next to **Contact points**.
+
+     - Select **+ New specific policy** to create a new policy, followed by 
+       **+ Add matcher** to choose the label and value from the alert
+       (for example `team = operator`). In this example, both alert1 and 
+       alert3 will be forwarded to the associated contact point. You can 
+       include multiple labels in a single notification policy.
+     - Choose the contact point designated to receive the alert notifications.
+     - Press **Save policy** to finalize your settings. Your new policy will
+       now be displayed in the list.
+
+5. **Configure Mute Timing and Silences**
+
+   To avoid receiving alerts during specific, recurring time periods, create a
```

Comment:
What would be a reason for this in manufacturing? e.g., the weekends as there are no shifts